src/promptflow-azure/promptflow/azure/_cli/entry.py

```diff
@@ -140,7 +140,11 @@ def main():
         version_devkit = get_promptflow_devkit_version()
         if version_devkit:
             version_dict["promptflow-devkit"] = version_devkit
-        return json.dumps(version_dict, ensure_ascii=False, indent=2, sort_keys=True, separators=(",", ": ")) + "\n"
+        version_dict_string = (
+            json.dumps(version_dict, ensure_ascii=False, indent=2, sort_keys=True, separators=(",", ": ")) + "\n"
+        )
+        print(version_dict_string)
```

Comment:
why previously no need print here?

Reply:
not sure, maybe forget change code here to align with pf.
I just update code here to aligh with pf